src/sections/Resume/Resume.jsx

```diff
@@ -0,0 +1,229 @@
+    marginLeft: styles.sizes.sidebarDimensions.width,
+    [theme.breakpoints.down("sm")]: {
+      marginLeft: 0
+    },
+    [theme.breakpoints.only("xl")]: {
+      marginLeft: styles.sizes.sidebarDimensions.maxWidth
+    }
+  },
+  resumeContent: {
+    width: styles.sizes.globalDimensions.percentageNineTenths,
+    maxWidth: styles.sizes.globalDimensions.sections.mediumSize,
+    margin: "0 auto",
+    padding: `${styles.sizes.thickness.thick21} 0 ${styles.sizes.thickness.thick21} 0`,
+    [theme.breakpoints.down("sm")]: {
+      paddingTop: styles.sizes.largeThickness.thick3,
+      paddingBottom: 0
+    }
+  },
+  information: {
+    display: "flex",
+    color: styles.colors.silver,
+    margin: `${styles.sizes.thickness.thick26} 0 30px 0`,
+    [theme.breakpoints.down("md")]: {
+      flexDirection: "column",
+      marginBottom: 0
+    }
+  },
+  timeline: {
+    width: styles.sizes.globalDimensions.percentageHalf,
+    padding: `0 ${styles.sizes.specialThick.spacingThick2}`,
+    [theme.breakpoints.down("md")]: {
+      width: styles.sizes.globalDimensions.percentageFull,
+      padding: "0"
+    },
+    [theme.breakpoints.down("sm")]: {
+      marginBottom: styles.sizes.thickness.thick11
+    }
+  },
+  subtitle: {
+    marginBottom: styles.sizes.thickness.thick21
+  },
+  skills: {
+    display: "flex",
+    [theme.breakpoints.down("md")]: {
+      flexDirection: "column"
+    }
+  },
+  skillCards: {
+    width: styles.sizes.globalDimensions.percentageHalf,
+    padding: `0 ${styles.sizes.specialThick.spacingThick2}`,
+    [theme.breakpoints.down("md")]: {
+      width: styles.sizes.globalDimensions.percentageFull,
+      padding: "0"
+    },
+    [theme.breakpoints.down("sm")]: {
+      marginBottom: styles.sizes.thickness.thick21
+    }
+  }
+}));
+
+const Resume = () => {
+  const classes = useStyles();
+  const [scrolling, setScrolling] = useState(false);
+  const [pageObject, setPageObject] = useState({});
+  const [animationEnd, setAnimationEnd] = useState(false);
+
+  const finishAnimation = () => {
+    setAnimationEnd(true);
+  };
+
+  const handleScroll = event => {
+    if (!animationEnd) {
+      const page = event.target;
+      setPageObject(page);
+      setScrolling(prev => !prev);
+    }
+  };
+
+  return (
+    <div
+      className={classes.resumeSection}
+      id='resumeSection'
+      onScroll={event => handleScroll(event)}
+    >
+      <div className={classes.resumeContent}>
+        <div>
+          <SectionTitle
+            title='My Resume.'
+            iconImage='text'
+            styling={{
+              fontFamily: styles.fontFamilies.Merriweather,
+              fontWeight: "bold"
+            }}
+          />
+        </div>
+        <div className={classes.information}>
+          <div className={classes.timeline}>
+            <div className={classes.subtitle}>
+              <SectionSubtitle subtitleText='EXPERIENCE' />
+            </div>
+            <TimelineMaker
+              timeline={[
```

Comment:
Create a trello card so that I can provide this information later

Reply:
Ready, sir!